src/Sentry.AspNetCore/HostingEnvironmentExtensions.cs

```diff
@@ -0,0 +1,18 @@
+#if NETSTANDARD2_0
+using IWebHostEnvironment = Microsoft.AspNetCore.Hosting.IHostingEnvironment;
+#else
+using Microsoft.AspNetCore.Hosting;
+#endif
+
+namespace Sentry.AspNetCore
+{
+    internal static class HostingEnvironmentExtensions
+    {
+        public static string RootPath(this IWebHostEnvironment env) => env.
+#if NETSTANDARD2_0
+            WebRootPath;
+#else
+            ContentRootPath;
```

Comment:
These are not the same paths... 🙈

Reply:
Lol, good catch, thanks @khellang
`WebRootPath` isn't there on the 3.0.0 version (it's the right namespace so I don't know what's wrong).

Reply:
You could try to do something [like this] (https://github.com/khellang/Middleware/blob/fc75a0402df3a941c7fd464af4a8c116257e3e4e/src/ProblemDetails/ProblemDetailsMiddleware.cs#L14-L18) to reduce churn 😊

Reply:
That looks better. Thanks for the tip.